pkg/daemon/ceph/osd/volume.go

```diff
@@ -69,7 +70,7 @@ func (a *OsdAgent) initializeDevices(context *clusterd.Context, devices *DeviceO
 		storeFlag = "--filestore"
 	}
 
-	baseArgs := []string{"lvm", "batch", "--prepare", storeFlag, "--yes"}
+	baseArgs := []string{"lvm", "batch", storeFlag}
```

Comment:
is there a case where `--prepare` and `--yes` are not passed now?

Reply:
I thought `--report` and `--prepare` are mutually exclusive. Tested it apparently I was wrong, thanks for the hint.